dev-requirements.txt

```diff
@@ -16,3 +16,12 @@ sphinx-rtd-theme
 # jupyter
 jupyter_server
 nest_asyncio
+
+eth-stdlib>=0.2.7,<0.3.0
+eth-abi>=5.1.0
+eth-typing>=4.1.0
+eth-utils>=4.1.0
+eth-account>=0.12.1
+vyper==0.3.10
+py-evm>=0.10.0b4
+rich
```

Comment:
This should not be duplicated here, these are part of pyproject.toml
```suggestion
```

Reply:
To be honest, I even did not see the .toml file and I even did not know that something like that exists. Now I know :)  I also did not 'pip install titanoboa' - my fault. 
However, with my understanding for now it should all move into the toml file, I will provide an update on that.

Reply:
Please review the latest commit